docs/UserGuide.md

```diff
@@ -346,7 +350,7 @@ Deletes a `Mastery Check` based on the `MASTERYCHECK_ID` you specify.
 
 ### 6.6 Viewing Information : `view`
 
-#### 6.6.1 Viewing a list of all Students : `-s`
+#### 6.6.1 Viewing a list of all `Students` : `-s`
```

Comment:
Actually do we want the headers to be in markdown as well? Cos currently the headers are not in markdown